### PR TITLE
Adding support for scan items

### PR DESCRIPTION
**Pull request by alejandroliu**
This *PR* adds support for importing/exporting SmartState Analysis Scan Profiles.

### CODE REVIEW

File: rhconsulting_scanitems.rake [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  def import_scan_items(import_dir)
+    Dir.glob("#{import_dir}/ScanProfile_*yaml") do |filename|
+      puts("Importing Scan Profile: #{File.basename(filename, '.yaml').gsub(/^ScanProfile_/, '')} ....")
+
+      hash = YAML.load_file(filename)
+      items = hash["definition"]
+      hash.delete("definition")
+      profile = ScanItemSet.find_by(:name => hash["name"]);
+      if profile.nil?
+	if hash["guid"].nil?
```
Comment (branic) [inline rhconsulting_scanitems.rake]: Indent this as well

File: rhconsulting_scanitems.rake [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	end
+	profile = ScanItemSet.new(hash)
+      else
+	profile.attributes = hash
+      end
+      profile.save!
+      
+      # Delete existing members
+      profile.members.each do |one|
+	if one.filename
```
Comment (branic) [inline rhconsulting_scanitems.rake]: Indent this as well

File: rhconsulting_scanitems.rake [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+	  # OK, this was defined through yaml file... just skip it
+	  next
+	  #profile.delete(one)
+	else
+	  one.destroy
+	end
+      end
+      items.each do |i|
+	if i['filename']
```
Comment (branic) [inline rhconsulting_scanitems.rake]: Here as well.

File: rhconsulting_scanitems.rake [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    import_scan_items(import_dir)
+  end
+
+private
+
+  def export_scan_items(export_dir, options)
+    ScanItemSet.all.each do |p|
+      # Skip read only entries
+      if p.read_only 
+	next
```
Comment (branic) [inline rhconsulting_scanitems.rake]: Please fix indentation so it is easier to read

**Comment by alejandroliu** [comment]
I have changed the indentation.  Is this better?

**Comment by branic** [comment]
Thanks @alejandroliu

**Pull request merged**